primitiv/devices/eigen/ops/flip.cc

```diff
@@ -0,0 +1,45 @@
+#include <primitiv/config.h>
+
+#include <primitiv/devices/eigen/device.h>
+#include <primitiv/devices/eigen/ops/common.h>
+
+namespace primitiv {
+namespace devices {
+
+void Eigen::flip_fw_impl(const Tensor &x, std::uint32_t dim, Tensor &y) {
```

Comment:
Maybe we can use `<Eigen matrix>.row/col_wise().reverse()` for the 0th and the last dimension.